modules/trusted-firmware-m/Kconfig

```diff
@@ -291,6 +291,18 @@ config TFM_PROFILE_TYPE_MINIMAL
 	  and reproduce the desired configuration through kconfig fragments.
 endchoice
 
+config TFM_CPU_FREQ_MHZ
+	int
+	default $(div, $(mul, 1, $(dt_node_int_prop_int,/cpus/cpu@0,clock-frequency)), 1000000) \
```

Comment:
Why a multiplication by 1?

Reply:
Great question. I got a type error when using div on the value from the dt_node, but not with mul. But when I tried it now to get the exact error it worked perfectly fine. So removed it.